test-e2e/android/app/src/main/java/com/mopro/mopro_app/MultiplierComponent.kt

```diff
@@ -49,8 +48,12 @@ fun MultiplierComponent() {
         ) { Text(text = "generate proof") }
         Button(
             onClick = {
+                val ethereumProof = uniffi.mopro.toEthereumProof(res.proof)
```

Comment:
These conversion looks weired to me. We definitely need to refine it sometime in the future

Reply:
https://github.com/zkmopro/mopro/issues/368
Hopefully will remove this when `CircomProof` is ready
It is used to check the `toEthereumProof` and `fromEthereumProof` works